Qt6.3 missing dependency

```
Run cmake -DCMAKE_BUILD_TYPE="Release" -DBUILD_SHARED_LIBS=ON -B "/home/runner/work/QXlsx/QXlsx/build" -S "QXlsx"
-- The CXX compiler identification is GNU 11.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE  
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libOpenGL.so   
-- Found WrapOpenGL: TRUE  
-- Could NOT find XKB (missing: XKB_LIBRARY XKB_INCLUDE_DIR) (Required is at least version "0.5.0")
-- Could NOT find WrapVulkanHeaders (missing: Vulkan_INCLUDE_DIR) 
Current Path of QXlsx
/home/runner/work/QXlsx/QXlsx/QXlsx/../
/home/runner/work/QXlsx/QXlsx/QXlsx/../QXlsx/header/
/home/runner/work/QXlsx/QXlsx/QXlsx/../QXlsx/source/
-- Configuring done
CMake Error at /home/runner/work/QXlsx/QXlsx/Qt/Qt/6.3.0/gcc_64/lib/cmake/Qt6Gui/Qt6GuiTargets.cmake:81 (set_target_properties):
-- Generating done
  The link interface of target "Qt6::GuiPrivate" contains:

    XKB::XKB

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  /home/runner/work/QXlsx/QXlsx/Qt/Qt/6.3.0/gcc_64/lib/cmake/Qt6Gui/Qt6GuiConfig.cmake:50 (include)
  /home/runner/work/QXlsx/QXlsx/Qt/Qt/6.3.0/gcc_64/lib/cmake/Qt6/Qt6Config.cmake:213 (find_package)
  CMakeLists.txt:17 (find_package)
```
Looks like libxkbcommon or libxkbcommon-x11 is missed. With Qt5 everything is ok

--- a/action/src/main.ts
+++ b/action/src/main.ts
@@ -243,7 +243,8 @@ const run = async (): Promise<void> => {
           "libxcb-xfixes0",
           "libxcb-xinerama0",
           "libxcb1",
-          "libxkbcommon-x11-0",
+          "libxkbcommon-dev",
+          "libxcb-xkb-dev",
         ].join(" ");
         const updateCommand = "apt-get update";
         const installCommand = `apt-get install ${dependencies} -y`;

